### PR TITLE
Clean up codebase and use flake8 instead of pylint

### PR DESCRIPTION
**Pull request by mprahl**
This is an attempt to make our codebase pep8 compliant. Also, instead of maintaining our own pylint script, I figured we could just use flake8 which works well out of the box and does more than pylint.

### CODE REVIEW

File: postmaster/utils.py [hunk 1]
```diff
@@ -197,62 +219,73 @@ def add_ldap_user_to_db(username, display_name):
...
+        log_msg = ('The following error occurred when adding an LDAP admin: '
+                   '{0}'.format(str(e)))
+        json_logger('error', username, log_msg)
+        ValidationError('A database error occurred. Please try again.')
     finally:
         db.session.close()
 
 
 def validate_wtforms_password(form, field):
-        """ Validates the password from a wtforms object
```
Comment (mprahl) [inline postmaster/utils.py]: This is shown as removed because the indentation was off.

File: postmaster/utils.py [hunk 2]
```diff
@@ -197,62 +219,73 @@ def add_ldap_user_to_db(username, display_name):
...
+                    error_msg = ('The user is currently locked out. Please '
+                                 'try logging in again later.')
+                    raise WtfStopValidation(error_msg)
+            log_msg = ('The administrator "{0}" entered an incorrect username '
+                       'or password'.format(username))
+            json_logger('auth', username, log_msg)
+            raise WtfStopValidation('The username or password was incorrect')
+        else:
+            ad_object = AD()
+            if ad_object.login(username, password) \
```
Comment (mprahl) [inline postmaster/utils.py]: This used to be two if statements, and I consolidated it to one.

**Comment by thatarchguy** [comment]
Codebase looks good. A robust library over a custom script is much better. I accidentally cause a conflict when I merged #188 first :/

**Comment by mprahl** [comment]
@thatarchguy merge conflict resolved.

**Comment by thatarchguy** [comment]
[LGTM](http://lgtm.in/i/Yu8lDUqWp)

[:+1:](http://lgtm.in/u/Yu8lDUqWp)

**Pull request merged**